### ISSUE TITLE
std::runtime error due to shape mismatch

### ISSUE THREAD

**Issue by ddubocan** [nanoporetech/dorado#97]
When trying to run dorado with `-x cpu` this is the error that occurs:

`dorado-0.1.1+eb48766-Linux/bin/dorado basecaller -x cpu  -r 6 dna_r10.4.1_e8.2_260bps_fast@v4.0.0/ pod5s/ `

```
terminate called after throwing an instance of 'std::runtime_error'
  what():  shape '[24, 6, -1, 5]' is invalid for input of size 36864
```

**Comment by vellamike** [comment]
Hi @ddubocan, does this happen with other models or only V4 fast?

**Comment by ddubocan** [comment]
Happens with other models as well.

**Comment by iiSeymour** [comment]
@ddubocan this is now fixed in v0.2.1.
